src/modules/bids-asks/helpers/select-order-book.js

```diff
@@ -95,7 +95,9 @@ const selectAggregatePricePoints = memoize((outcomeID, side, orders, orderCancel
       const obj = {
         isOfCurrentUser: shareCountPerPrice[price].isOfCurrentUser,
         shares: formatShares(shareCountPerPrice[price].shares),
-        price: formatEtherTokens(parseFloat(price))
+        price: formatEtherTokens(parseFloat(price)),
+        sharesEscrowed: formatShares(shareCountPerPrice[price].sharesEscrowed),
+        tokensEscrowed: formatEtherTokens(parseFloat(shareCountPerPrice[price].tokensEscrowed)),
```

Comment:
Same comment as above, re: avoid JS numbers.  Why is `parseFloat` applied to `tokensEscrowed` but not to `sharesEscrowed`?  Are the formatters set up differently?  Recommend consistency if possible.